install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -11,67 +11,64 @@
 toc::[]
 
 == Overview
-You can provision your OpenShift cluster with
-link:../../architecture/additional_concepts/storage.html[persistent storage] using
-https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Storage_Administration_Guide/ch-nfs.html[NFS].
-Some familiarity with Kubernetes and NFS is assumed.
 
-The Kubernetes
-link:../../architecture/additional_concepts/storage.html[persistent volume]
-framework allows administrators to provision a cluster with persistent storage
-and gives users a way to request those resources without having any knowledge of
-the underlying infrastructure. Persistent volumes are not bound to a single
-project or namespace; they can be shared across the OpenShift cluster.
-link:../../architecture/additional_concepts/storage.html#persistent-volume-claims[Persistent
-volume claims], however, are specific to a project or namespace and can be
-requested by users.
-
-For a detailed example, see the guide for
-https://github.com/openshift/origin/tree/master/examples/wordpress[WordPress and
-MySQL using NFS].
-
-[IMPORTANT]
+OpenShift clusters can be provisioned with
+link:../../architecture/additional_concepts/storage.html[persistent storage]
+using NFS. Persistent volumes (PVs) and persistent volume claims (PVCs) provide
+a convenient method for sharing a volume across a project. While the
+NFS-specific information contained in a PV definition could also be defined
+directly in a pod definition, doing so does not create the volume as a distinct
+cluster resource, making the volume more susceptible to conflicts.
+
+This topic covers the specifics of using the NFS persistent storage type. Some
+familiarity with OpenShift and
+https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Storage_Administration_Guide/ch-nfs.html[NFS]
+is beneficial. See the
+link:../../architecture/additional_concepts/storage.html[Persistent Storage]
+architecture topic for details on the OpenShift persistent volume (PV) framework
+in general.
+
+[NOTE]
 ====
-High-availability of storage in the infrastructure is left to the underlying
-storage provider.
+_Project_ and _namespace_ are used interchangeably throughout this document. See
+link:../../architecture/core_concepts/projects_and_users.html#namespaces[Projects
+and Users] for details on the relationship.
```

Comment:
Alex,

--all-namespaces is an oc command line option, so we expose namespace to users.
Also, "namespace" appears in some command output and in certain error messages.
My point being that we can't entirely hide the word "namespace", and that is
the reason I used both namespace and project in the docs, and why I have the note.
It's your call I just want you to know my motivation in using both.

jeff

----- Original Message -----
> >  ====
> > -High-availability of storage in the infrastructure is left to the
> > underlying
> > -storage provider.
> > +_Project_ and _namespace_ are used interchangeably throughout this
> > document. See
> > +link:../../architecture/core_concepts/projects_and_users.html#namespaces[Projects
> > +and Users] for details on the relationship.
> 
> @jeffvance Is there any reason I couldn't just update any references of
> "namespace" to "project" so we can lose this Note?
> 
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/openshift-docs/pull/1726/files/9ce51e49014fad81a65119371df97a3505615842#r56526732